be1-go/channel/lao/mod.go

```diff
@@ -107,6 +110,28 @@ func NewChannel(channelID string, hub channel.HubFunctionalities, msg message.Me
 		attendees:       make(map[string]struct{}),
 		log:             log,
 	}
+
+	newChannel.registry = newChannel.NewLAORegistry()
+
+	return newChannel
+}
+
+// NewLAORegistry creates a new registry for the LAO channel
+func (c *Channel) NewLAORegistry() registry.MessageRegistry {
+	registry := registry.NewMessageRegistry()
+
+	registry.Register(messagedata.LaoState{}, c.processLaoState)
+	registry.Register(messagedata.LaoUpdate{}, c.processEmptyFun)
+	registry.Register(messagedata.MeetingCreate{}, c.processEmptyFun)
+	registry.Register(messagedata.MeetingState{}, c.processEmptyFun)
+	registry.Register(messagedata.MessageWitness{}, c.processMessageObject)
```

Comment:
would make more sense, no ? 
```suggestion
	registry.Register(messagedata.MessageWitness{}, c.processMessageWitness)
```

Reply:
yes, will do the update